File: tests/Integration/jtethan.js

jcond {
    typeAonly: jsys.tag == "typeA";
    fogonly: jsys.type == "fog";
}

let count = 10;

jtask function remoteEchoReceiver(str, count) {
    console.log("Received an echo!");
    console.log(str, count);
}

let long = jsys.long;
let lat = jsys.lat;

setInterval(()=> {
    remoteEcho("Device to worker communication");

    console.log("Calling compyou.. ");

}, 3000);

File: tests/Integration/jtethan.c
void remoteEchoReceiver(char *s, int count);


jtask void remoteEcho(char* s) {
    static int count = 0;
    
    remoteEchoReceiver(s, count++);
}

int main(int argc, char *argv[])
{
 //   localme(10, "cxxxxyyyy");
    return 0;
}
